install/operator/pkg/syndesis/action/install.go

```diff
@@ -160,6 +160,12 @@ func (a *installAction) Execute(ctx context.Context, syndesis *synapi.Syndesis)
 
 	resourcesThatShouldExist[syndesisRoute.GetUID()] = true
 
+	// sets the openshift management console URL
+	config.SetOpenshiftManagementConsoleUrl(ctx, a.clientTools)
```

Comment:
Might you wanna check this is only for openshift?

Reply:
But syndesis only works in openshift, right ?